slicedimage/backends/_caching.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import os
+import io
+
+from diskcache import Cache
+from io import BytesIO
```

Comment:
this import should be grouped with stdlib imports.  we're following the PEP8 convention (more specifically the smarkets variant described [here](https://github.com/PyCQA/flake8-import-order#styles))

generally, i also avoid importing the same module twice, so I would just use `io.BytesIO` in this module.